lib/rules/jsx-no-literals.js

```diff
@@ -41,7 +42,7 @@ module.exports = {
     function reportLiteralNode(node) {
       context.report({
         node: node,
-        message: message
+        message: `${message}:${sourceCode.getText(node).trim()}`
```

Comment:
Let’s add a space after the colon, and it’d be good to have some kind of boundary markers around the text - backticks or curly quotes, maybe?

Separately, I’m not sure if the trimming is a good idea - if it’s a literal, the whitespace could be important.

Reply:
> Let’s add a space after the colon, and it’d be good to have some kind of boundary markers around the text - backticks or curly quotes, maybe?

Sure, let's use it like this->   : "TEXT"

> Separately, I’m not sure if the trimming is a good idea - if it’s a literal, the whitespace could be important.

I think Not having the trimming really impact the readability of the output, in every case that I have work with this kind of output, I have needed to use the trimming. So, I woud like to keep it.